### PR TITLE
Create Pie Charts section (#336)

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Adresses #336.

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/pie.md [hunk 1]
```diff
@@ -1,3 +1,249 @@
...
+| [*values*](../chart.md#p-values)            | Slice values  |  |
+| [*marker*](../chart.md#p-marker)    | dictionary  | `colors` can be defined as an array of color names or values. |
+| [*options*](../chart.md#p-options)  | dictionary  | `textinfo` can be set to customize how texts are displayed (see [*textinfo* reference](https://plotly.com/javascript/reference/#pie-textinfo) for details).  |
+| [*layout*](../chart.md#p-layout)  | dictionary  | `showlegend` can be set to False to hide the legend.<br/>`annotations` can hold text annotations that are displayed in the chart control.  |
+
+## Examples
+
+### Simple pie chart {data-source="gui:doc/examples/charts/pie-simple"}
+
+Here this example, we want to represent how much area is covered
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/pie.md]: Here => in?

File: docs/manuals/gui/viselements/charts/pie.md [hunk 2]
```diff
@@ -1,3 +1,249 @@
...
+by forest in countries in the world. Only the first few countries are
+represented since too many would make the chart unreadable.
+
+Our data set, where the area is provided as thousands of hectares in 2020,
+comes from [FAO](https://www.fao.org) and is defined as follows:
+
+```py
+data = pd.DataFrame({
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/pie.md]: A tiny dataframe sometimes...

File: docs/manuals/gui/viselements/charts/pie.md [hunk 3]
```diff
@@ -1,3 +1,249 @@
...
+
+And the chart definition is the following:
+
+!!! example "Page content"
+
+    === "Markdown"
+
+        ```
+        <|{data}|chart|type=pie|values=Area|labels=Country|>
+        ```
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/pie.md:50]: Missing layout?

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/manuals/gui/viselements/charts/pie.md [hunk 4]
```diff
@@ -1,3 +1,249 @@
...
+| [*marker*](../chart.md#p-marker)    | dictionary  | `colors` can be defined as an array of color names or values. |
+| [*options*](../chart.md#p-options)  | dictionary  | `textinfo` can be set to customize how texts are displayed (see [*textinfo* reference](https://plotly.com/javascript/reference/#pie-textinfo) for details).  |
+| [*layout*](../chart.md#p-layout)  | dictionary  | `showlegend` can be set to False to hide the legend.<br/>`annotations` can hold text annotations that are displayed in the chart control.  |
+
+## Examples
+
+### Simple pie chart {data-source="gui:doc/examples/charts/pie-simple"}
+
+Here this example, we want to represent how much area is covered
+by forest in countries in the world. Only the first few countries are
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/pie.md]: in different countries around the world

File: docs/manuals/gui/viselements/charts/pie.md [hunk 5]
```diff
@@ -1,3 +1,249 @@
...
+represented since too many would make the chart unreadable.
+
+Our data set, where the area is provided as thousands of hectares in 2020,
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/pie.md]: A bit unclear to me

File: docs/manuals/gui/viselements/charts/pie.md [hunk 6]
```diff
@@ -1,3 +1,249 @@
...
+
+<figure>
+    <img src="../pie-simple-d.png" class="visible-dark" />
+    <img src="../pie-simple-l.png" class="visible-light"/>
+    <figcaption>Simple pie chart</figcaption>
+</figure>
+
+### Styling {data-source="gui:doc/examples/charts/pie-styling"}
+
+You can specify what the color of each slice will be and these sorts of
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/pie.md]: Remove "and these sort of things"

**Review by dinhlongviolin1 — APPROVED** [review]
Just a few comments

File: docs/manuals/gui/viselements/charts/pie.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/pie.md]: 👍

File: docs/manuals/gui/viselements/charts/pie.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/pie.md:50]: Not really: it's useless. Dropped if from the HTML format.
Thx

File: docs/manuals/gui/viselements/charts/pie.md (hunk 4 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/pie.md]: 👌

File: docs/manuals/gui/viselements/charts/pie.md (hunk 5 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/pie.md]: Rephrased for clarity.

File: docs/manuals/gui/viselements/charts/pie.md (hunk 6 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/pie.md]: Done.

File: docs/manuals/gui/viselements/charts/pie.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/pie.md]: Dropped!

**Pull request merged**